.zygote/gateway.toml

```diff
@@ -0,0 +1,16 @@
+# [nodeset.gateway]
+# in = "main"
+
+# [[nodepool.kvm.main.tcp.port]]
```

Comment:
Use `function.gateway.tcp.port`.
The port number on VM is always set to function.gateway.tcp.port.source